services/static-webserver/client/source/class/osparc/ErrorPage.js

```diff
@@ -72,56 +79,12 @@ qx.Class.define("osparc.ErrorPage", {
   },
 
   statics: {
-    FRIENDLY_HTTP_STATUS: {
-      "200": "OK",
-      "201": "Created",
-      "202": "Accepted",
-      "203": "Non-Authoritative Information",
-      "204": "No Content",
-      "205": "Reset Content",
-      "206": "Partial Content",
-      "300": "Multiple Choices",
-      "301": "Moved Permanently",
-      "302": "Found",
-      "303": "See Other",
-      "304": "Not Modified",
-      "305": "Use Proxy",
-      "306": "Unused",
-      "307": "Temporary Redirect",
-      "400": "Bad Request",
-      "401": "Unauthorized",
-      "402": "Payment Required",
-      "403": "Forbidden",
-      "404": "Not Found",
-      "405": "Method Not Allowed",
-      "406": "Not Acceptable",
-      "407": "Proxy Authentication Required",
-      "408": "Request Timeout",
-      "409": "Conflict",
-      "410": "Gone",
-      "411": "Length Required",
-      "412": "Precondition Required",
-      "413": "Request Entry Too Large",
-      "414": "Request-URI Too Long",
-      "415": "Unsupported Media Type",
-      "416": "Requested Range Not Satisfiable",
-      "417": "Expectation Failed",
-      "418": "I'm a teapot",
-      "429": "Too Many Requests",
-      "500": "Internal Server Error",
-      "501": "Not Implemented",
-      "502": "Bad Gateway",
-      "503": "Service Unavailable",
-      "504": "Gateway Timeout",
-      "505": "HTTP Version Not Supported"
-    },
 
     POS: {
       LOGO: 0,
       PANDA: 1,
-      ERROR: 3,
```

Comment:
Just some thoughts, no need to answer. ☺️ 

If I understand this well, we're getting rid of those "friendly http status". Should we keep them somewhere in case they might be needed again? I also understand it doesn't make sense to keep code here if it's not used anymore.

I imagine they were anyway redundant if we always have our custom "MESSAGES". Do we always have them?

Reply:
> If I understand this well, we're getting rid of those "friendly http status". Should we keep them somewhere in case they might be needed again? I also understand it doesn't make sense to keep code here if it's not used anymore.

I would use version control to "revert" it if it happens we need it again

> I imagine they were anyway redundant if we always have our custom "MESSAGES". Do we always have them?

The errors to this "error page" are always provided by the backend but it would not harm to have a default in case the backend delivers an status error w/o a message. Let's see what @ignapas @jsaq007 think about it and how to implement it.

Reply:
We always have these here in the git repo but yes I think it's not the worst idea to have the error stored and handled in a good way in the exceptional cases. I would just log the error in the console or in the worst case show a flash message for the user so they are aware that something didn't quite go right. (On a need basis as most are trivial and only distract the user. Perhaps we could think of a concept for when and what the user needs/would like to see?

Reply:
@jsaq007  can you then follow up on that in a different PR or you think that this should rather be included here? The main purpose here is to satisfy POs request and get the issue close so I would suggest the first solution